packages/integration-tests/lambda.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const { lambda } = require('@cumulus/common/aws');
+const AWS = require('aws-sdk');
+
+/**
+ * Recursively gets all pages for an AWS lambda Aliases/Version object query
+ *
+ * @param {Object} config - A AWS Lambda query configuration object
+ * @param {string} key - Name of the object type being queried.  'Versions' or 'Aliases'
+ * @param {Object} listFunction - The paginated AWS function we want to get all 'pages' from
+ * @returns {Promise.Object[]} Returns a concatenated list of all the objects
+ *                             returned from this page and every page following.
+ */
+async function getAllPages(config, key, listFunction) {
+  const lambdaConfig = Object.assign({}, config);
+  const page = await listFunction(lambdaConfig).promise();
+  if (!page.NextMarker) {
+    return page[key];
+  }
+  const pages = page[key];
+  lambdaConfig.Marker = page.NextMarker;
+
+  return pages.concat(await getAllPages(lambdaConfig, key, listFunction.promise()));
+}
+
+/**
+ * Takes a lambda function name and returns all aliases for that function
+ *
+ * @param {string} lambdaFunctionName - The name of a lambda function
+ * @returns {Promise.Object[]} returns the promise of a list of AWS Lambda Alias objects
+ */
+async function getLambdaAliases(lambdaFunctionName) {
```

Comment:
I think here and in the next function it would help to describe what alias and version objects are.

Reply:
@laurenfrederick   Would either a link to the AWS lambda docs, or just referring to it as an AWS Lambda Alias/Version work?

Reply:
Maybe just a short description of what is version vs alias. Like version is 1, 2, 3, 4 (right?) alias is alias name? Or maybe this is just confusing for me and you can ignore me.

Reply:
Sort of.   The response from something like getLambdaVersions is a list of version objects (https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Lambda.html#listVersionsByFunction-property), and  is roughly the same thing as a functionConfigurationObject from the AWS API ..... so I suppose it's not properly a lambda version object. (?) 

The Alias response returns a list of alias objects (https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Lambda.html#listVersionsByFunction-property) 

I could include the above links in the function header if that seems helpful.

Reply:
Added refs to the SDK call in the jsdoc : 22308985